setup.py

```diff
@@ -44,11 +44,11 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket ~= 1.27",
-        "qiskit ~= 1.0",
+        "pytket ~= 1.29",
+        "qiskit ~= 1.1",
         "qiskit-algorithms ~= 0.3.0",
-        "qiskit-ibm-runtime ~= 0.23.0",
-        "qiskit-aer ~= 0.14.0",
+        "qiskit-ibm-runtime ~= 0.24.1",
+        "qiskit-aer ~= 0.14.2",
         "numpy",
```

Comment:
Ideally we'd have "< 2" here but that's already implied by pytket's requirements, so no need.

Reply:
I have added that. We might update pytket in the future, but we probably will face additional changes here.